crates/amaru-ledger/src/rules/transaction/fees.rs

```diff
@@ -19,6 +19,11 @@ use amaru_kernel::{HasLovelace, Lovelace, MintedTransactionOutput, TransactionIn
 pub enum InvalidFees {
     #[error("unknown collateral input at position {position}")]
     UnknownCollateralInput { position: usize },
+    #[error("Collateral return value is greater than total collateral input (input: {total_collateral_input}, return: {total_collateral_return})")]
```

Comment:
```suggestion
    #[error("collateral return value ({total_collateral_return}) is greater than total collateral input ({total_collateral_input})")]
```

1. Start with a lowercase, so that we can more easily chain error messages: `2nd transaction failed: invalid fees: collateral return...`. Although, we should really think about producing _structured_ errors in the long run. 

2. Embed information where it's relevant; avoid repetition and makes the error message easier to read.